cmd/soroban-cli/tests/fixtures/test-wasms/custom_type/src/lib.rs

```diff
@@ -77,8 +71,8 @@ impl Contract {
         complex
     }
 
-    pub fn address(_env: Env, address: Address) -> Address {
-        address
+    pub fn addresse(_env: Env, addresse: Address) -> Address {
```

Comment:
typo?

Reply:
Nope an update from the sdk added a new implicit `address` method. Should I add a comment?